pkgs/top-level/all-packages.nix

```diff
@@ -10408,6 +10408,7 @@ in
 
   gleam = callPackage ../development/compilers/gleam {
     inherit (darwin.apple_sdk.frameworks) Security;
+    inherit (darwin) libiconv;
```

Comment:
```suggestion
```

not required.

Reply:
Do you mean it is not required in this file?

I had a failure with libiconv missing on the bot update PR, and so I just added it here. I verified locally that the failure was due to that lib missing. Do you mean to say that it works without this on your machine?

Reply:
@SuperSandro2000 Sorry to bother you with this, I need more details here.

Reply:
> Do you mean it is not required in this file?

It shouldn't be required.

Reply:
Yeah, it is not required.

Reply:
Ok I'll know for the future, thanks!